cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -51,6 +51,8 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.String("log-driver", "", "specify a log driver")
 	flags.StringSlice("log-opt", nil, "log driver options, as key value pairs")
+
+	flags.Int64("pids-limit", -1, "set container pid limit")
```

Comment:
The actual API uses the string `max` for unlimited. `0` is a valid limit, means you may not create any more processes.